### PR TITLE
Add nav bar pl

### PR DESCRIPTION
**Pull request by peyton-lalli**

### CODE REVIEW

**Comment by peyton-lalli** [comment]
Please review so I can move on with the nav bar, thanks

File: src/App.vue [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-        </v-col>
-      </v-row>
-    </v-container>
+      <v-toolbar-title 
+        color = "mediumBlue"
+        class = "font-weight-bold">MUSIC DEPARTMENT</v-toolbar-title>
+      <v-spacer></v-spacer>
+      <v-toolbar-items class="hidden-sm-and-down">
+      </v-toolbar-items>
+    </v-app-bar>
```
Comment (ethanimooney) [inline src/App.vue:19]: I would make this a component - sent you what we did on the last project.

**Review by ethanimooney — CHANGES_REQUESTED** [review]

**Comment by peyton-lalli** [comment]
Can merge pls?

**Review by ethanimooney — APPROVED** [review]
LGTM - will go through and prettify later

**Pull request merged**